components/webdriver_server/lib.rs

```diff
@@ -347,6 +410,131 @@ impl<'de> Visitor<'de> for TupleVecMapVisitor {
     }
 }
 
+fn compute_tick_duration(tick_actions: &ActionSequence) -> u64 {
+    let mut duration = 0;
+    match &tick_actions.actions {
+        ActionsType::Null { actions } => {
+            for action in actions.iter() {
+                let NullActionItem::General(GeneralAction::Pause(pause_action)) = action;
+                duration = cmp::max(duration, pause_action.duration.unwrap_or(0));
+            }
+        },
+        ActionsType::Pointer {
+            parameters: _,
+            actions,
+        } => {
+            for action in actions.iter() {
+                if let PointerActionItem::General(GeneralAction::Pause(pause_action)) = action {
+                    duration = cmp::max(duration, pause_action.duration.unwrap_or(0));
+                } else if let PointerActionItem::Pointer(PointerAction::Move(move_action)) = action
+                {
+                    duration = cmp::max(duration, move_action.duration.unwrap_or(0));
+                }
+            }
+        },
+        _ => (),
```

Comment:
It would be good to be explicit about ignoring key actions here, so if any more action types are introduced in the future then we get an error about not handling them.